src/Microsoft.AspNet.Mvc.Core/Rendering/Html/HtmlHelper.cs

```diff
@@ -1016,5 +1045,22 @@ protected virtual string GenerateValue(string expression, object value, string f
         {
             return _htmlGenerator.GetClientValidationRules(ViewContext, modelExplorer, expression);
         }
+
+        protected virtual IEnumerable<SelectListItem> GetSelectList([NotNull] ModelMetadata metadata)
```

Comment:
hmm, think I named this before decided to close #602.  without `bool?`, I can't think of another type this method could support.  I'll rename all the overloads to `GetEnumSelectList`.